### PR TITLE
[INFRA-747] Add support for path-based routing while attaching to an existing ALB

### PR DESCRIPTION
**Pull request by pgAdmin**

### CODE REVIEW

File: test/templates/expected_service_with_env_alb_template_path_based.yml [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+      TaskDefinition: !Ref 'DummyTaskDefinition'
+    Type: AWS::ECS::Service
+  DummyListenerRule:
+    Properties:
+      Actions:
+        - TargetGroupArn: !Ref 'TargetGroupDummy'
+          Type: forward
+      Conditions:
+        - Field: path-pattern
+          PathPatternConfig:
```
Comment (pgAdmin) [inline test/templates/expected_service_with_env_alb_template_path_based.yml:87]: Review Pointer: 
This is the actual change in CF Template.

File: test/deployment/service_template_generator_test.py [hunk 2]
```diff
@@ -323,6 +323,97 @@ def mock_describe_rules(ListenerArn=None, Marker=None):
...
+            "services": {
+                "Dummy": {
+                    "memory_reservation": Decimal(1000),
+                    "command": None,
+                    "secrets_name": "something",
+                    "http_interface": {
+                        "internal": False,
+                        "alb": {
+                            "create_new": False,
+                            "path": "/api/*"
```
Comment (pgAdmin) [inline test/deployment/service_template_generator_test.py:354]: Review Pointer

File: cloudlift/version/__init__.py [hunk 3]
```diff
@@ -1 +1 @@
-VERSION = '1.13.1'
+VERSION = '1.13.2'
```
Comment (pgAdmin) [inline cloudlift/version/__init__.py:1]: Updating sub-version since looks like a minor enhancement.

**Review by aswinkarthik — APPROVED** [review]

File: cloudlift/config/service_configuration.py [hunk 4]
```diff
@@ -164,6 +164,9 @@ def _validate_changes(self, configuration):
                                 "host": {
                                     "type": "string"
                                 },
+                                "path": {
+                                    "type": "string"
+                                },
                                 "priority": {
```
Comment (siliconsenthil) [inline cloudlift/config/service_configuration.py:170]: Use "oneOf" jsonschema to mandate at least one of 'host' or 'path'

**Review by siliconsenthil — APPROVED** [review]

File: cloudlift/config/service_configuration.py (hunk 4 above)
Comment (pgAdmin) [reply cloudlift/config/service_configuration.py:170]: oneOf can only be applied on presence of keys, not their values.

**Pull request merged**